.mailmap

```diff
@@ -0,0 +1,17 @@
+Cenk Gündoğan <mail-github@cgundogan.de> <cnkgndgn@gmail.com>
+Cenk Gündoğan <mail-github@cgundogan.de> <mail@cgundogan.de>
+Hauke Petersen <devel@haukepetersen.de> <hauke.petersen@fu-berlin.de>
+Hauke Petersen <devel@haukepetersen.de> <mail@haukepetersen.de>
+Joakim Nohlgård <joakim.nohlgard@eistec.se> <joakim.gebart@eistec.se>
+Joakim Nohlgård <joakim.nohlgard@eistec.se> <joakim@gebart.se>
+Kaspar Schleiser <kaspar@schleiser.de>
+Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de> <ludwig.ortmann@fu-berlin.de>
+Martine Lenders <mail@martine-lenders.eu> <authmillenon@gmail.com>
+Martine Lenders <mail@martine-lenders.eu> <m.lenders@fu-berlin.de>
+Martine Lenders <mail@martine-lenders.eu> <mail@martin-lenders.de>
+Martine Lenders <mail@martine-lenders.eu> <mlenders@inf.fu-berlin.de>
```

Comment:
I would prefer to use `m.lenders@fu-berlin.de` as my "primary" mail address for RIOT